.pre-commit-config.yaml

```diff
@@ -11,6 +11,14 @@ repos:
         stages: [commit]
         types: [python]
 
+      - id: black-jupyter
+        name: black-jupyter
+        entry: black
+        language: python
```

Comment:
I'm not quite sure of the lineage of `language: system` for the other pre-commit hooks, but we should _probably_ be consistent?

Reply:
Yeah I tried it with `system` and it didn't like that plus the additional dependencies field. I have never seen `language: system` before, but I was hesitant about changing all the other pre-commit actions.

Reply:
Hmm...okay no worries. Per https://github.com/stac-utils/pystac/pull/1034#issuecomment-1468032168, I might be taking some time soon to write up some standards docs for this sort of stuff (e.g. **pystac-client** doesn't use `language: system`), which will be a chance to revisit how we set up the **pre-commit** hooks. So I think it's fine to do it the way that works now, and then we can bring everything in line later when we hit all the repos. Thanks.